backend/app/src/main/java/com/moh/phlat/backend/service/ProcessDataServiceImpl.java

```diff
@@ -24,4 +27,91 @@ public List<ProcessData> getProcessDataWithMessages(Long controlTableId, String
         }
 
     }
+
+    public List<ReportSummary> getReportSummary(Long controlTableId) {
+		String rowStatus;
+		Long rcount;
+
+		List<ReportSummary> items = new ArrayList<ReportSummary>();
+
+		rowStatus ="TOTAL INPUT RECORDS";
```

Comment:
the variable names and the values assigned are not matching. I guess the name should be reportAttributeName and reportAttributeValue

Reply:
I updated the code to use your suggested variable names.